benchmark/streams/writable-manywrites.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common');
+const Writable = require('stream').Writable;
+
+const bench = common.createBenchmark(main, {
+  n: [2e6]
+});
+
+function main(conf) {
+  const n = +conf.n;
+  const b = new Buffer(1024);
```

Comment:
If this is going to be kept as `new Buffer(...)`, can you add a comment explaining why (e.g. to allow comparison with older Node.js versions). That said, I'd prefer this to be changed to `Buffer.alloc(...)` given that Node.js 4 is the lowest currently supported version and the new constructors have been backported to that version already.